test/unit/browser/json_schema/validation_test.dart

```diff
@@ -48,7 +48,7 @@ final Logger _logger = new Logger('test_validation');
 void main([List<String> args]) {
   configureJsonSchemaForBrowser();
 
-  if (args?.isEmpty ?? false) {
+  if (args?.isEmpty != true) {
```

Comment:
This change reverses the logic that was in place before, is that intended?

Reply:
good catch!